src/helix-ui/icons/exclamation-triangle.svg

```diff
@@ -1,3 +1,3 @@
 <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
-  <path d="M9.19 9.008c-.01.275-.072.612-.14.75-.067.137-.622.25-.896.25h-.189c-.274 0-.614-.05-.752-.113-.14-.062-.269-.612-.277-.887l-.12-4c-.008-.276.032-.613.09-.75.058-.138.605-.25.88-.25h.584c.275 0 .616.045.758.1.14.055.216.624.207.9l-.146 4zm-.19 3c0 .274-.05.612-.113.75-.062.137-.612.25-.886.25-.276 0-.613-.05-.75-.113-.139-.062-.25-.613-.25-.887 0-.275.05-.613.111-.75.063-.138.612-.25.889-.25.274 0 .612.05.75.112.136.062.25.613.25.888zm5.832 1.292c-.28-.562-.567-1.12-.85-1.679-1.683-3.315-3.366-6.63-5.046-9.947-.204-.402-.49-.667-.925-.674-.423-.007-.724.234-.924.627-1.982 3.906-3.965 7.812-5.94 11.724a1.395 1.395 0 0 0-.145.663c.03.595.476.984 1.084.984 1.97.003 3.942.001 5.913.001 1.962 0 3.926.002 5.89 0 .928-.002 1.38-.819.944-1.699z"/>
+  <path d="M14.832 13.3c.438.88-.016 1.697-.944 1.698C11.925 15.001 9.962 15 8 15c-1.973 0-3.944.002-5.914 0-.608 0-1.054-.39-1.084-.985a1.395 1.395 0 0 1 .145-.663C3.123 9.44 5.104 5.533 7.088 1.627c.2-.393.5-.634.925-.627.433.007.72.272.923.674 1.68 3.317 3.364 6.632 5.046 9.947.284.56.57 1.117.851 1.68zm-6.332.2c.275 0 .5-.225.5-.5v-1c0-.275-.225-.5-.5-.5h-1a.5.5 0 0 0-.5.5v1a.5.5 0 0 0 .5.5h1zm.065-3c.268 0 .52-.249.507-.5l.175-5c.032-.299-.203-.5-.472-.5h-1.55a.48.48 0 0 0-.475.5l.148 5c.008.275.235.5.504.5h1.163z"/>
```

Comment:
🎨 Exclamation point was really rough. Corrected and normalized across all `exclamation-*` icons.